pandas/core/arrays/arrow/array.py

```diff
@@ -120,19 +120,63 @@ def cast_for_truediv(
             # Intentionally not using arrow_array.cast because it could be a scalar
             # value in reflected case, and safe=False only added to
             # scalar cast in pyarrow 13.
-            return pc.cast(arrow_array, pa.float64(), safe=False)
-        return arrow_array
+            # In arrow, common type between integral and float64 is float64,
```

Comment:
Could you trim this comment and the one above with just a link to https://github.com/apache/arrow/issues/35563 and `# GH: 56645`. While the comments are helpful, too many are distracting from the code

Reply:
Yea, cleaned up the comments.